apps/meteor/ee/server/lib/ldap/Manager.ts

```diff
@@ -223,11 +224,18 @@ export class LDAPEEManager extends LDAPManager {
 			const userFields = ensureArray<string>(fieldMap[ldapField]);
 
 			for await (const userField of userFields) {
-				const [roleId] = userField.split(/\.(.+)/);
-				allowedRoles.push(roleId);
+				const [roleName] = userField.split(/\.(.+)/);
+
+				const role = roles.find((role) => role.name === roleName);
```

Comment:
```suggestion
				const [roleIrOrName] = userField.split(/\.(.+)/);

				const role = roles.find((role) => role._id === roleIrOrName) ?? roles.find((role) => role.name === roleIrOrName);
```

Accept both ids and names in the LDAP setting, prioritizing IDs.
This way it'll ensure that any environment currently configuring with IDs will not need to be reconfigured.

Reply:
updated!